<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>View</class>
 <widget class="QWidget" name="View">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>784</width>
    <height>602</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>200</height>
   </size>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,1">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="toolBarSystemLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="toolBarLayout" stretch="0,0,0,0">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="navigationFrame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QToolButton" name="backHistoryButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string notr="true">Back</string>
            </property>
            <property name="icon">
             <iconset resource="../../../3rdparty/icons.qrc">
              <normaloff>:/icons/backup/go-previous-view.svg</normaloff>:/icons/backup/go-previous-view.svg</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="forwardHistoryButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string notr="true">Forward</string>
            </property>
            <property name="icon">
             <iconset resource="../../../3rdparty/icons.qrc">
              <normaloff>:/icons/backup/go-next-view.svg</normaloff>:/icons/backup/go-next-view.svg</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="toolBarPlaceHolder" native="true">
         <layout class="QHBoxLayout" name="toolBarHolder">
          <property name="spacing">
           <number>0</number>
          </property>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="viewControlsFrame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QToolButton" name="splitToolButton">
            <property name="text">
             <string>view</string>
            </property>
            <property name="icon">
             <iconset resource="../../../3rdparty/icons.qrc">
              <normaloff>:/icons/backup/view-split-left-right.svg</normaloff>:/icons/backup/view-split-left-right.svg</iconset>
            </property>
            <property name="popupMode">
             <enum>QToolButton::InstantPopup</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="closeToolButton">
            <property name="text">
             <string>close</string>
            </property>
            <property name="icon">
             <iconset resource="../../../3rdparty/icons.qrc">
              <normaloff>:/icons/backup/view-close.svg</normaloff>:/icons/backup/view-close.svg</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="titleFrame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="titleLabel">
            <property name="text">
             <string notr="true"/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="centralLayout"/>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../3rdparty/icons.qrc"/>
 </resources>
 <connections/>
</ui>
